apiary.apib

```diff
@@ -145,6 +145,17 @@ and suggester data.
 
 + Response 204
 
+## Project data management [/projects/{project}/data]
+
+### delete project index data [DELETE]
+
+This will delete project index data but keep the project in web application configuration so that
+can be indexed from scratch.
+The project data (whatever is the project's directory under source root) will be left intact.
+This will also mark the project as not indexed.
```

Comment:
what is the benefit of keeping it in the configuration when you need to reindex it from scratch?

Reply:
I needed this multiple times, e.g. when indexer failed for some reason or when the repository was force pushed.

Reply:
But can't you just delete the whole project and not just the data?

Reply:
Sure, this is just a little bit less invasive.